target/snitch_cluster/sw/runtime/banshee/src/snrt.h

```diff
@@ -25,9 +26,12 @@
 #include "dm.h"
 #include "dma.h"
 #include "eu.h"
+#include "kmp.h"
+#include "omp.h"
 #include "perf_cnt.h"
 #include "printf.h"
 #include "riscv.h"
+// #include "snitch_cluster_global_interrupts.h" # what is Banshee equivalent?
```

Comment:
Can we resolve this before merging?

Reply:
This will need a discussion about the Banshee features with @SamuelRiedel I am not sure if Banshee currently has an interrupt model in Banshee.

Reply:
Okay, but if there is no Banshee equivalent then we can just remove this line